pkg/apis/build/v1alpha1/build_pod.go

```diff
@@ -84,6 +86,16 @@ var (
 		MountPath: "/builderPullSecrets",
 		ReadOnly:  true,
 	}
+	notaryV1Volume = corev1.VolumeMount{
+		Name:      notaryDirName,
+		MountPath: "/var/notary/v1",
+		ReadOnly:  true,
+	}
+	reportVolume = corev1.VolumeMount{
+		Name:      reportDirName,
+		MountPath: "/var/report",
+		ReadOnly:  false,
```

Comment:
can this be read only if it just contains report.toml

Reply:
it has to be writable because the export step writes to it.